docker-compose.yml

```diff
@@ -0,0 +1,262 @@
+version: "3.4"
+
+x-common-variables: &common-build-variables
+  # Trenger auth for mvn clean install, er mest sannsynlig i $HOME/.m2/settings.xml, klipp og lim inn. (password er token)
+  github_username: FYLL_INN_GITHUB_BRUKER
```

Comment:
Hvis dere exporter disse i eks `.zprofile` , så er det mulig å sette de med https://docs.docker.com/compose/environment-variables/
```
github_username: "${GITHUB_USERNAME}"
```

Reply:
Ja, men jeg tenkte det var greiest å ikke be noen om å endre noen filer som ikke er i repoet

Reply:
Endte med å sette dem i .env i dette repoet, da docker-compose leser den filen by default. :)